diff -Naur inkscape-1.1.2_2022-02-04_0a00cf5339.orig/src/extension/internal/pdfinput/svg-builder.cpp inkscape-1.1.2_2022-02-04_0a00cf5339/src/extension/internal/pdfinput/svg-builder.cpp
--- inkscape-1.1.2_2022-02-04_0a00cf5339.orig/src/extension/internal/pdfinput/svg-builder.cpp	2022-02-04 20:21:47.000000000 -0500
+++ inkscape-1.1.2_2022-02-04_0a00cf5339/src/extension/internal/pdfinput/svg-builder.cpp	2022-08-05 13:27:01.374066328 -0400
@@ -89,7 +89,7 @@
 
     // Set default preference settings
     _preferences = _xml_doc->createElement("svgbuilder:prefs");
-    _preferences->setAttribute("embedImages", "1");
+    _preferences->setAttribute("embedImages", "0"); // JDR: changed default
     _preferences->setAttribute("localFonts", "1");
 }
 
@@ -1032,6 +1032,7 @@
     } else {
         _font_specification = "Arial";
     }
+    char *orig = g_strdup(_font_specification); // JDR
 
     // Prune the font name to get the correct font family name
     // In a PDF font names can look like this: IONIPB+MetaPlusBold-Italic
@@ -1040,6 +1041,7 @@
     char *font_style_lowercase = nullptr;
     const char *plus_sign = strstr(_font_specification, "+");
     if (plus_sign) {
+        *(orig + (plus_sign - _font_specification)) = ' '; // JDR
         font_family = g_strdup(plus_sign + 1);
         _font_specification = plus_sign + 1;
     } else {
@@ -1053,6 +1055,7 @@
         style_delim[0] = 0;
     }
 
+#if 0 // JDR
     // Font family
     if (font->getFamily()) { // if font family is explicitly given use it.
         sp_repr_css_set_property(_font_style, "font-family", font->getFamily()->getCString());
@@ -1066,6 +1069,9 @@
             sp_repr_css_set_property(_font_style, "font-family", font_family);
         }
     }
+#endif // JDR
+    sp_repr_css_set_property(_font_style, "font-family", orig); // JDR
+    g_free(orig);
 
     // Font style
     if (font->isItalic()) {
@@ -1322,10 +1328,11 @@
 
                 // Set style and unref SPCSSAttr if it won't be needed anymore
                 // assume all <tspan> nodes in a <text> node share the same style
-                sp_repr_css_change(text_node, glyph.style, "style");
-                if ( glyph.style_changed && i != _glyphs.begin() ) {    // Free previous style
-                    sp_repr_css_attr_unref((*prev_iterator).style);
-                }
+                // JDR: this is a bad assumption.  Don't do that.
+                sp_repr_css_change(tspan_node, glyph.style, "style");
+                // if ( glyph.style_changed && i != _glyphs.begin() ) {    // Free previous style
+                //    sp_repr_css_attr_unref((*prev_iterator).style);
+                //}
             }
             new_tspan = false;
         }
@@ -1521,7 +1528,7 @@
         return nullptr;
     }
     // Decide whether we should embed this image
-    int attr_value = 1;
+    int attr_value = 0;
     sp_repr_get_int(_preferences, "embedImages", &attr_value);
     bool embed_image = ( attr_value != 0 );
     // Set read/write functions
